src/v/model/tests/model_serialization_test.cc

```diff
@@ -60,9 +60,11 @@ SEASTAR_THREAD_TEST_CASE(tristate_rt_test) {
     tristate<size_t> empty(std::nullopt);
     tristate<size_t> present(1024);
 
+    // NOLINTBEGIN(performance-move-const-arg)
```

Comment:
What's the rationale for keeping the moves?

Reply:
Compilation fails if you don't pass an r-value reference. I think Felix has addressed this, but it's not yet in the tree.